### PR TITLE
UI modification for sponsor buttons

### PR DESCRIPTION
**Pull request by maximdudai**
Before:
image

After:
image

### CODE REVIEW

File: src/images/logos/github-white.png
Comment (manojVivek) [inline src/images/logos/github-white.png:1]: The cut on the top and bottom feels strange, can you please swap it with the one that has the full circle?

**Review by manojVivek — CHANGES_REQUESTED** [review]
Just a minor change request, looks great otherwise!

File: src/images/logos/github-white.png
Comment (maximdudai) [reply src/images/logos/github-white.png:1]: I don't know why I didn't realize right away that it didn't look decent 🤣, but it's fixed now 👍

**Comment by maximdudai** [comment]
I made a small mistake 🫠 the branch was accidentally deleted by me before it was merged. I've just reopened the pull request to enable merging. I apologize for any inconvenience this may have caused. :)

**Review by manojVivek — APPROVED** [review]
Looks great! Thank you for your contribution!

**Pull request merged**